### PR TITLE
Add Katy & Robert to list of maintainers in readme

### PR DESCRIPTION
**Pull request by gonuke**
Apologies for accidentally pushing to master with small changes on the website!  I'll refrain from that....

This change will add @katyhuff and @FlanFlanagan to the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,4 +181,6 @@ Feedstock Maintainers
 * [@bam241](https://github.com/bam241/)
 * [@gonuke](https://github.com/gonuke/)
 * [@scopatz](https://github.com/scopatz/)
+* [@katyhuff](https://github.com/katyhuff/)
+* [@FlanFlanagan](https://github.com/FlanFlanagan/)
```
Comment (scopatz) [inline README.md:185]: This is not correct.  They should be added to the `recipe/meta.yaml` file, and only there.

**Comment by scopatz** [comment]
Thanks for putting these in @gonuke

**Comment by gonuke** [comment]
I see....

Since I already (accidentally) pushed the necessary changes, I'll withdraw this.

**Comment by scopatz** [comment]
No worries! Thanks for taking this over

**Comment by gonuke** [comment]
But maybe because I did not PR the changes, the automatic changes to `README` didn't happen??

**Pull request closed without merging**